src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentProvider.cs

```diff
@@ -63,20 +64,24 @@ public async Task GenerateDocumentationProposalAsync(DocumentationCommentSnippet
 
             // Do not do IntelliCode line completions if we're about to generate a documentation comment
             // so that won't have interfering grey text.
-            var intellicodeLineCompletionsDisposable = await _suggestionManager!.DisableProviderAsync(SuggestionServiceNames.IntelliCodeLineCompletions, cancellationToken).ConfigureAwait(false);
+            var intelliCodeLineCompletionsDisposable = await _suggestionManager!.DisableProviderAsync(SuggestionServiceNames.IntelliCodeLineCompletions, cancellationToken).ConfigureAwait(false);
```

Comment:
how do yo know _suggestionManager is not null here?

Reply:
Chcked above with the Enabled check.

Reply:
nullable annotate the Enabled check so that the compiler knows that _sugestionManager is non null if it returns true.